Clinical-Genomics/cg
# Upgrade to Python 3.9

## Description
This PR upgrades cg to use Python 3.9. See [migration guide](https://docs.python.org/3/whatsnew/3.9.html#porting-to-python-3-9). We are not affected by any of the breaking changes. All tests pass locally.

Blocked by https://github.com/Clinical-Genomics/cg/pull/2556 since the current flask version we use (1.3) only officially supports Python 3.8. Also need to upgrade housekeeper to 3.9 before merging since they are executed in the same conda environment as cg.

## Testing
- Upgrade the Python version used in `S_cg`
- Deploy this branch to stage in Hasta and on the vm.
- Verify that the cli works and orders can be placed

### Fixed
- Upgrade to Python 3.9 from 3.8

### This [version](https://semver.org/) is a
